WeWatch/WeWatch/UICommons/MovieCard.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  MovieCard.swift
+//  WeWatch
+//
+//  Created by Anton on 26/12/2024.
+//
+
+import SwiftUI
+
+internal struct MovieCard: View {
+    
+    private let bookmark: Bool
```

Comment:
Boolean variables are best named using a format like “isBookmarked” for clarity and consistency.